pkg/controllers/podgroup_controller_test.go

```diff
@@ -159,34 +160,36 @@ func Test_Run(t *testing.T) {
 		t.Run(c.name, func(t *testing.T) {
 			controller, kClient := setUp(ctx, c.podNames, c.pgName, c.podPhase, c.minMember, c.previousPhase, c.podGroupCreateTime, nil)
 			// 0 means not set
+			ps := makePods(c.podNames, c.pgName, c.podPhase, nil)
```

Comment:
this line should be in front of L162, as the comment is describing `len(c.podNextPhase) != 0`